integrationtest/wildfly/pom.xml

```diff
@@ -30,6 +30,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <byteman.agent.path>${org.jboss.byteman:byteman:jar}</byteman.agent.path>
         <byteman.script.path>${basedir}${file.separator}src${file.separator}test${file.separator}resources${file.separator}disablejpadapters.btm</byteman.script.path>
+        <hibernate.search.module.slot>main</hibernate.search.module.slot>
```

Comment:
Careful with that property name: we use the same name in another module, not for testing but to generate module aliases (I think).
Maybe that's all the more reason to use qualified property names? `test.hibernate.search.module.slot`, or `it.hibernate.search.module.slot` here?